javaparser-core-testing/src/test/java/com/github/javaparser/javadoc/JavadocTest.java

```diff
@@ -166,7 +180,7 @@ void issue1533() {
         List<JavadocDescriptionElement> elements = compilationUnit.getType(0).getJavadoc().get().getDescription().getElements();
         assertEquals(3, elements.size());
         assertEquals(new JavadocSnippet("hallo "), elements.get(0));
-        assertEquals(new JavadocInlineTag("link", LINK, " Foo"), elements.get(1));
+        assertEquals(new JavadocInlineTag("link", LINK, "Foo"), elements.get(1));
```

Comment:
I am not sure here. What if we had a tab o three spaces before Foo? Would they be relevant or not?

Reply:
As far as I can see it's only relevant for printing. I should to check though.